optimade/server/main.py

```diff
@@ -45,6 +45,7 @@
     LOGGER.info("DEBUG MODE")
 
 app = FastAPI(
+    root_path=CONFIG.root_path,
```

Comment:
Could you also add this line in the same place for the index meta-database please?

https://github.com/Materials-Consortia/optimade-python-tools/blob/e20407ddfa467cc036632f7415eddadbb4bbc000/optimade/server/main_index.py#L38-L50

Reply:
Sure. I just pushed the suggested changes.